mobile/src/viewModels/HomeViewModel.ts

```diff
@@ -97,6 +103,38 @@ export default class HomeViewModel {
             })) || EmptyArr;
         }
 
+        if (AppViewModel.Instance.QOL.isUnfinished) {
+            return [
+                <IFinishQolTipItem>{
+                    id: 'finish-qol',
+                    type: 'finish-qol',
+                    title: 'Tap to continue your QoL Survey!',
+                },
+                // Currently always shows 'daily' check-in as this feature has not yet been implemented
+                <ICheckInTipItem>{
+                    id: 'check-in',
+                    type: 'check-in',
+                    title: AppViewModel.Instance.CreateCheckIn.question || 'Create a new check-in!',
+                },
```

Comment:
Rather than repeat this code block 3 times, it might be better to initialize an empty array for the return value and simply append to it once at the very end before returning that value, adding the other parts in an `if-else if` statement beforehand.

Reply:
Thats the solution I implemented in PR #79